release/ray_release/byod/build.py

```diff
@@ -1,11 +1,53 @@
 from typing import List
 
-from ray_release.test import Test
+import boto3
+import subprocess
+
+from ray_release.logger import logger
+from ray_release.test import (
+    Test,
+    S3_BUCKET,
+    DATAPLANE_FILENAME,
+)
 
 
 def build_anyscale_byod_images(tests: List[Test]) -> None:
     """
     Builds the Anyscale BYOD images for the given tests.
     """
-    # TODO(aslonnie): Build anyscale byod images for the given ray images
+    s3 = boto3.client("s3")
+    s3.download_file(
+        Bucket=S3_BUCKET,
+        Key=DATAPLANE_FILENAME,
+        Filename=DATAPLANE_FILENAME,
```

Comment:
could you also check in the sha256 hash (in source code) and check it after the downloading?

Reply:
I'm not sure what this means; will poke you on Monday!

Reply:
meaning adding a:

````
want_digest = "abc123..."
with open(build_file, "rb") as f:
    digest = hashlib.file_digest(f, "sha256")
if digest != want_digest:
    raise Exception()
````